create-integration-deployment/action.yml

```diff
@@ -42,30 +51,31 @@ runs:
       working-directory: ./integration-deployments
       run: |
           git reset --hard
-          git checkout -b deploy-${{ inputs.integrationName }}-${{ inputs.version }}
-          git push origin deploy-${{ inputs.integrationName }}-${{ inputs.version }}
+          git checkout -b deploy-${{ inputs.graphProjectName }}-${{ inputs.version }}
+          git push origin deploy-${{ inputs.graphProjectName }}-${{ inputs.version }}
     - name: Bump version in package.json and commit changes
       shell: bash
       working-directory: ./integration-deployments
       run: |
           echo "//registry.npmjs.org/:_authToken=${{ inputs.npmAuthToken }}" > .npmrc
           cd ./src/${{ inputs.integrationName }}
           yarn
-          yarn upgrade @jupiterone/graph-${{ inputs.integrationName }}@^${{ inputs.version }}
+          yarn upgrade @jupiterone/graph-${{ inputs.graphProjectName }}@^${{ inputs.version }}
           git add .
-          git commit -m "bump ${{ inputs.integrationName }} to version ${{ inputs.version }} deploy2dev"
+          git commit -m "bump ${{ inputs.graphProjectName }} to version ${{ inputs.version }} deploy2dev"
     - name: Create Pull Request
       id: create_pull_request
       uses: peter-evans/create-pull-request@v4
       with:
         token: ${{ inputs.githubToken }}
         path: ./integration-deployments
-        branch: deploy-${{ inputs.integrationName }}-${{ inputs.version }}
+        branch: deploy-${{ inputs.graphProjectName }}-${{ inputs.version }}
         base: ${{ inputs.mainBranch }}
-        title: "[Github Action] - Deploy graph-${{ inputs.integrationName }} v${{ inputs.version }}"
+        title: "[Github Action] - Deploy graph-${{ inputs.graphProjectName }} v${{ inputs.version }}"
         body: |
           ## Summary
+          ${{ inputs.releaseNotes }}
```

Comment:
Adds in the release notes generated by Auto. Example notes: https://github.com/JupiterOne/graph-azure/releases/tag/v5.40.9

Reply:
Unbelievable. This is awesome.